hack/deploy-setup.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# This script inits a cluster to allow cluster-logging-operator
+# to deploy logging.  It assumes it is capable of login as a
+# user who has the cluster-admin role
+
+set -e
+
+source "$(dirname $0)/common"
+
+repo_dir="$(dirname $0)/.."
+ELASTICSEARCH_OP_REPO=${ELASTICSEARCH_OP_REPO:-${repo_dir}/../elasticsearch-operator}
+
+load_manifest() {
+  local repo=$1
+  local namespace=${2:-}
+  if [ -n "${namespace}" ] ; then
+    namespace="-n ${namespace}"
+  fi
+
+  pushd ${repo}/manifests
+    for m in $(ls); do
+      if [ "$(echo ${EXCLUSIONS[@]} | grep -o ${m} | wc -w)" == "0" ] ; then
+        oc create -f ${m} ${namespace:-}
+      fi
+    done
+  popd
+}
+
+load_manifest ${repo_dir}
+load_manifest ${ELASTICSEARCH_OP_REPO} openshift-logging
```

Comment:
es needs a namespace but not clo?

Reply:
I believe this was to try and satisfy the desire to keep the elasticsearch-operator generic allowing us to load all the resources into a given namespace where CLO is expected to already exist in the desired namespace by defining it in the manifest